### PR TITLE
chore(ui): servers_view folder

### PR DESCRIPTION
**Pull request by daniellacosse**
while migrating the first component to pure lit i found myself moving some folders around - I thought I would break that off into a prerequisite PR.

This PR does the following:
- replaces `lit-html` with `lit` (recommended)
- creates a `servers_view` folder which contains the original servers_view as well as the `server_list`, `server_card` and `server_connection_indicator` (was `server_connection_viz`)
- puts `servers_view` in a new `views` folder
- points the storybook at `views`, specifically.

Next I'll migrate everything in the `views` folder in-place to lit 2.0. This way we have a clear taxonomy of view to presentational component, and a separation between old code and new.

### CODE REVIEW

File: src/www/app/app.ts [hunk 1]
```diff
@@ -15,8 +15,8 @@
 import * as errors from "../model/errors";
 import * as events from "../model/events";
 import {Server} from "../model/server";
-import {ServerConnectionState} from "../ui_components/server_card";
-import {ServerListItem} from "../ui_components/server_list";
+import {ServerConnectionState} from "../views/servers_view/server_connection_indicator";
```
Comment (daniellacosse) [inline src/www/app/app.ts:18]: I could re-export these from `servers_view` if that's amenable

File: src/www/views/servers_view/server_connection_indicator/stories.ts [hunk 2]
```diff
@@ -15,13 +15,15 @@
   limitations under the License.
 */
 
-import {html} from "lit-html";
+import "./index";
```
Comment (daniellacosse) [inline src/www/views/servers_view/server_connection_indicator/stories.ts:18]: I won't have to do this anymore once we move to lit

File: src/www/app/app.ts (hunk 1 above)
Comment (daniellacosse) [reply src/www/app/app.ts:18]: so it'd be

```
import {ServerListItem, ServerConnectionState} from "../views/servers_view";
```

File: src/www/views/servers_view/server_connection_indicator/index.ts [hunk 3]
```diff
@@ -24,8 +24,8 @@ export enum ServerConnectionState {
   DISCONNECTED = "DISCONNECTED",
 }
 
-@customElement("server-connection-viz")
-export class ServerConnectionViz extends LegacyElementMixin(PolymerElement) {
+@customElement("server-connection-indicator")
```
Comment (fortuna) [inline src/www/views/servers_view/server_connection_indicator/index.ts:27]: I like the new name!

File: src/www/.storybook/localize.ts [hunk 4]
```diff
@@ -25,6 +25,8 @@ type FormattableMessage =
   | FormatXMLElementFn<symbol | object, string | symbol | object | (string | symbol | object)[]>;
 
 export const localize = (messageID: string, ...formatKeyValueList: FormattableMessage[]): string => {
+  if (!messageID) return "";
```
Comment (fortuna) [inline src/www/.storybook/localize.ts:28]: This means we will fail silently if messages are missing.
Should we output something else, like "MISSING_MESSAGE"?

We should at least call console.warn

File: src/www/app/app.ts (hunk 1 above)
Comment (fortuna) [reply src/www/app/app.ts:18]: That would be fine with me. We are leaking implementation details and the app shouldn't need to know how the servers list view is implemented.

**Review by fortuna — APPROVED** [review]

File: src/www/.storybook/localize.ts (hunk 4 above)
Comment (daniellacosse) [reply src/www/.storybook/localize.ts:28]: Actually, there are instances where a messageID is just not provided. For example, errorMessageId is a required property on `ServerListItem`. If an empty string is given, we'd see `MISSING_MESSAGE` everywhere there's an error, even if there's no message to show.

You know what - I'll just fix the bug at the source

File: src/www/app/app.ts (hunk 1 above)
Comment (daniellacosse) [reply src/www/app/app.ts:18]: Right, basically my thinking here

File: src/www/views/servers_view/server_list/index.ts [hunk 5]
```diff
@@ -55,7 +55,7 @@ export class ServerList extends PolymerElement {
       <template is="dom-repeat" items="[[servers]]">
         <server-card
           disabled="[[item.errorMessageId]]"
-          error-message="[[localize(item.errorMessageId)]]"
+          error-message="[[getErrorMessage(item.errorMessageId)]]"
```
Comment (fortuna) [inline src/www/views/servers_view/server_list/index.ts:58]: With Lit you could just inline the logic in the template.

Comment (daniellacosse) [reply src/www/views/servers_view/server_list/index.ts:58]: Yep - that's next!

**Review by fortuna — APPROVED** [review]

**Pull request merged**